[gamma:x64-windows] build failure

**Host Environment**
- OS: Windows 10 Pro 64-bit Version 2004 build 19041.508 with Windows Feature Experience Pack 120.2212.31.0
- Compiler: Microsoft Visual Studio 2019 Version 16.7.4

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install gamma:x64-windows --debug`

**Failure logs** 
```
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Using vcpkg-root: F:\GitHub\vcpkg
[DEBUG] Using installed-root: F:\GitHub\vcpkg\installed
[DEBUG] Using buildtrees-root: F:\GitHub\vcpkg\buildtrees
[DEBUG] Using downloads-root: F:\GitHub\vcpkg\downloads
[DEBUG] Using packages-root: F:\GitHub\vcpkg\packages
[DEBUG] Using scripts-root: F:\GitHub\vcpkg\scripts
[DEBUG] Default binary cache path is: C:\Users\jimmy\AppData\Local\vcpkg\archives
Computing installation plan...
[DEBUG] Found path: C:\Program Files\CMake\bin/cmake.exe
[DEBUG] CreateProcessW("F:\GitHub\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    43536 us
[DEBUG] CreateProcessW("F:\GitHub\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DVCPKG_ROOT_DIR=F:/GitHub/vcpkg -DPACKAGES_DIR=F:/GitHub/vcpkg/packages -DBUILDTREES_DIR=F:/GitHub/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=F:/GitHub/vcpkg/installed -DDOWNLOADS=F:/GitHub/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "F:\GitHub\vcpkg\buildtrees\0.vcpkg_tags.cmake")
[DEBUG] cmd_execute_and_stream_data() returned 0 after    37586 us
The following packages will be built and installed:
    gamma[core]:x64-windows
[DEBUG] CreateProcessW("C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -all -prerelease -legacy -products * -format xml)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    39079 us
Detecting compiler hash for triplet x64-windows...
[DEBUG] Found path: C:\Program Files\Git\cmd/git.exe
[DEBUG] Found path: C:\Program Files\Git\bin/git.exe
[DEBUG] CreateProcessW("F:\GitHub\vcpkg\downloads\tools\git-2.26.2-1-windows\mingw32\bin\git.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    31916 us
[DEBUG] CreateProcessW(cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" -vcvars_ver=14.2 amd64   2>&1 <NUL" & echo cdARN4xjKueKScMy9C6H& set)
[DEBUG] cmd_execute_and_stream_data() returned 0 after  1209868 us
[DEBUG] CreateProcessW("F:\GitHub\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DCURRENT_PORT_DIR=F:/GitHub/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=F:/GitHub/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=F:/GitHub/vcpkg/packages/detect_compiler_x64-windows -DCMD=BUILD -DTARGET_TRIPLET=x64-windows "-DTARGET_TRIPLET_FILE=F:\GitHub\vcpkg\triplets\x64-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=F:/GitHub/vcpkg/downloads -DVCPKG_CONCURRENCY=17 -DGIT=F:/GitHub/vcpkg/downloads/tools/git-2.26.2-1-windows/mingw32/bin/git.exe -DVCPKG_ROOT_DIR=F:/GitHub/vcpkg -DPACKAGES_DIR=F:/GitHub/vcpkg/packages -DBUILDTREES_DIR=F:/GitHub/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=F:/GitHub/vcpkg/installed -DDOWNLOADS=F:/GitHub/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "F:\GitHub\vcpkg\scripts\ports.cmake")
[DEBUG] -- Configuring x64-windows
[DEBUG] [1/1] cmd /c "cd .. && "F:/GitHub/vcpkg/downloads/tools/cmake-3.17.2-windows/cmake-3.17.2-win32-x86/bin/cmake.exe" "F:/GitHub/vcpkg/scripts/detect_compiler" "-DCMAKE_MAKE_PROGRAM=F:/GitHub/vcpkg/downloads/tools/ninja/1.10.0-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=F:/GitHub/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=F:/GitHub/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=F:/GitHub/vcpkg" "-D_VCPKG_INSTALLED_DIR=F:/GitHub/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=F:/GitHub/vcpkg/packages/detect_compiler_x64-windows""
[DEBUG] #COMPILER_HASH#12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG] -- Configuring done
[DEBUG] -- Generating done
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG] 
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     VCPKG_TARGET_ARCHITECTURE
[DEBUG] 
[DEBUG] 
[DEBUG] -- Build files have been written to: F:/GitHub/vcpkg/buildtrees/detect_compiler/x64-windows-rel
[DEBUG] 
[DEBUG] cmd_execute_and_stream_data() returned 0 after  3741389 us
[DEBUG] 
[DEBUG] Detecting compiler hash for triplet x64-windows: 12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG] <abientries for gamma:x64-windows>
[DEBUG]   CMakeLists.txt|57d5e34ed184e6bac48aacbf8ab0dbf18e0dd3bc
[DEBUG]   CONTROL|ffcf02c65041d28ba3da340f18f0427de5ed1246
[DEBUG]   cmake|3.17.2
[DEBUG]   features|core
[DEBUG]   libsndfile|e7fe0010b75cfd4d8059dc45672e0d250e93b9ae
[DEBUG]   portaudio|550f6fa8e30e561d25c43b413d8830f7e0e8fa78
[DEBUG]   portfile.cmake|d0b9f3c61710128e509c16bec0cca8f4d8eca47b
[DEBUG]   post_build_checks|2
[DEBUG]   powershell|2e410fc5e429fc6ad478976eff3efcffe9d53a0620c7c31c2b8bfe1f0976cade74632fc202bf58959cec111f03b824ba42ad4d74c8a66d67ec21bbfe9b9c626d
[DEBUG]   triplet|3eab5c64d9cf628fc0aec277fde66dbdc69b8516-a1c0eabb0c5177b6a8fb97a58ae398880c47b352-12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG]   vcpkg_check_linkage|dc5fd80fa0a6f18c7eabee6f4da803b5abdeebcb
[DEBUG]   vcpkg_configure_cmake|d9e525769b8519c26e4c0446242434961527d33b
[DEBUG]   vcpkg_copy_pdbs|4bbc15802daf8f9f8a92794c10b0a89cfca1e742
[DEBUG]   vcpkg_from_git|4834edbcec7eed8d12c6b1ad3c25af63d5fa6586
[DEBUG]   vcpkg_from_github|4624dffdf2b8d85ffdfde0850369fa96abdf0bb8
[DEBUG]   vcpkg_install_cmake|75d1079d6f563d87d08c8c5f63b359377aa19a6e
[DEBUG] </abientries>
Starting package 1/1: gamma:x64-windows
Building package gamma[core]:x64-windows...
Could not locate cached archive: C:\Users\jimmy\AppData\Local\vcpkg\archives\15\15beee9111c33bf3b3f76680dc12a2e03cfd35b9.zip
[DEBUG] CreateProcessW("F:\GitHub\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DPORT=gamma -DCURRENT_PORT_DIR=F:/GitHub/vcpkg/ports/gamma -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_NO_DOWNLOADS=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -DFEATURES=core -DALL_FEATURES= -DCMD=BUILD -DTARGET_TRIPLET=x64-windows "-DTARGET_TRIPLET_FILE=F:\GitHub\vcpkg\triplets\x64-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=F:/GitHub/vcpkg/downloads -DVCPKG_CONCURRENCY=17 -DGIT=F:/GitHub/vcpkg/downloads/tools/git-2.26.2-1-windows/mingw32/bin/git.exe -DVCPKG_ROOT_DIR=F:/GitHub/vcpkg -DPACKAGES_DIR=F:/GitHub/vcpkg/packages -DBUILDTREES_DIR=F:/GitHub/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=F:/GitHub/vcpkg/installed -DDOWNLOADS=F:/GitHub/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "F:\GitHub\vcpkg\scripts\ports.cmake")
-- Note: gamma only supports static library linkage. Building static library.
-- Using cached F:/GitHub/vcpkg/downloads/LancePutnam-Gamma-cc442ad0c5da369966cd937a96925c7b9a04e9e5.tar.gz
-- Cleaning sources at F:/GitHub/vcpkg/buildtrees/gamma/src/7b9a04e9e5-b457a996be.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source F:/GitHub/vcpkg/downloads/LancePutnam-Gamma-cc442ad0c5da369966cd937a96925c7b9a04e9e5.tar.gz
-- Using source at F:/GitHub/vcpkg/buildtrees/gamma/src/7b9a04e9e5-b457a996be.clean
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: ninja -v
    Working Directory: F:/GitHub/vcpkg/buildtrees/gamma/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      [F:\GitHub\vcpkg\buildtrees\gamma\config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/5272900/config-x64-windows-out.log)


Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:305 (vcpkg_execute_required_process)
  ports/gamma/portfile.cmake:17 (vcpkg_configure_cmake)
  scripts/ports.cmake:79 (include)


[DEBUG] cmd_execute_and_stream_data() returned 1 after  8817976 us
Error: Building package gamma:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: gamma:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
[DEBUG] F:\GitHub\vcpkg\toolsrc\src\vcpkg\install.cpp(493)
[DEBUG] Exiting after 14186821 us (14175818 us)

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.
```

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBSNDFILE_LIB
    linked by target "gamma" in directory F:/GitHub/vcpkg/buildtrees/gamma/src/7b9a04e9e5-b457a996be.clean

-- Generating done
```

@JackBoosY Thank you for your reply. I ran `.\vcpkg.exe remove libsndfile:x64-windows --recurse` to remove libsndfile package first and tried to install gamma:x64-windows again. The error still happened like below.

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBSNDFILE_LIB
```

** Failure Log**
> .\vcpkg.exe install gamma:x64-windows
Computing installation plan...
The following packages will be built and installed:
    gamma[core]:x64-windows
  * libsndfile[core,external-libs]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/2: libsndfile:x64-windows
Building package libsndfile[core,external-libs]:x64-windows...
Using cached binary package: C:\Users\jimmy\AppData\Local\vcpkg\archives\e7\e7fe0010b75cfd4d8059dc45672e0d250e93b9ae.zip
Building package libsndfile[core,external-libs]:x64-windows... done
Installing package libsndfile[core,external-libs]:x64-windows...
Installing package libsndfile[core,external-libs]:x64-windows... done
Elapsed time for package libsndfile:x64-windows: 390.1 ms
Starting package 2/2: gamma:x64-windows
Building package gamma[core]:x64-windows...
Could not locate cached archive: C:\Users\jimmy\AppData\Local\vcpkg\archives\15\15beee9111c33bf3b3f76680dc12a2e03cfd35b9.zip
-- Note: gamma only supports static library linkage. Building static library.
-- Using cached F:/GitHub/vcpkg/downloads/LancePutnam-Gamma-cc442ad0c5da369966cd937a96925c7b9a04e9e5.tar.gz
-- Cleaning sources at F:/GitHub/vcpkg/buildtrees/gamma/src/7b9a04e9e5-b457a996be.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source F:/GitHub/vcpkg/downloads/LancePutnam-Gamma-cc442ad0c5da369966cd937a96925c7b9a04e9e5.tar.gz
-- Using source at F:/GitHub/vcpkg/buildtrees/gamma/src/7b9a04e9e5-b457a996be.clean
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: ninja -v
    Working Directory: F:/GitHub/vcpkg/buildtrees/gamma/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      [F:\GitHub\vcpkg\buildtrees\gamma\config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/5273046/config-x64-windows-out.log)


Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:305 (vcpkg_execute_required_process)
  ports/gamma/portfile.cmake:17 (vcpkg_configure_cmake)
  scripts/ports.cmake:79 (include)


Error: Building package gamma:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: gamma:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

--- a/ports/gamma/CMakeLists.txt
+++ b/ports/gamma/CMakeLists.txt
@@ -3,8 +3,7 @@ cmake_minimum_required (VERSION 3.0)
 project (gamma)
 
 # dependent on libsndfile and portaudio
-find_path(LIBSNDFILE_H sndfile.h)
-find_library(LIBSNDFILE_LIB NAMES libsndfile-1 libsndfile)
+find_package(SndFile CONFIG REQUIRED)
 find_path(PORTAUDIO_H portaudio.h)
 find_library(PORTAUDIO_LIB NAMES portaudio)
 
@@ -25,10 +24,10 @@ set(SOURCEFILES
         src/Timer.cpp
         src/SoundFile.cpp)
 
-include_directories(. Gamma ${CMAKE_INSTALL_FULL_INCLUDEDIR} ${PORTAUDIO_H} ${LIBSNDFILE_H})
+include_directories(. Gamma ${CMAKE_INSTALL_FULL_INCLUDEDIR} ${PORTAUDIO_H})
 
 add_library (gamma ${SOURCEFILES})
-target_link_libraries(gamma PUBLIC ${LIBSNDFILE_LIB} ${PORTAUDIO_LIB})
+target_link_libraries(gamma PUBLIC SndFile::sndfile ${PORTAUDIO_LIB})
 
 install(
   TARGETS gamma

--- a/ports/gamma/CONTROL
+++ b/ports/gamma/CONTROL
@@ -1,4 +1,5 @@
 Source: gamma
-Version: gamma-2018-01-27-1
+Version: gamma-2018-01-27
+Port-Version: 2
 Build-Depends: libsndfile, portaudio
 Description: Gamma is a cross-platform, C++ library for doing generic synthesis and filtering of signals. It is oriented towards real-time sound and graphics applications, but is equally useful for non-real-time tasks. Gamma is designed to be "light-footed" in terms of memory and processing making it highly suitable for plug-in development or embedding in other C++ projects.

--- a/ports/gamma/CONTROL
+++ b/ports/gamma/CONTROL
@@ -1,5 +1,6 @@
 Source: gamma
 Version: gamma-2018-01-27
 Port-Version: 2
+Homepage: https://github.com/LancePutnam/Gamma
 Build-Depends: libsndfile, portaudio
 Description: Gamma is a cross-platform, C++ library for doing generic synthesis and filtering of signals. It is oriented towards real-time sound and graphics applications, but is equally useful for non-real-time tasks. Gamma is designed to be "light-footed" in terms of memory and processing making it highly suitable for plug-in development or embedding in other C++ projects.

--- a/ports/gamma/portfile.cmake
+++ b/ports/gamma/portfile.cmake
@@ -1,5 +1,3 @@
-include(vcpkg_common_functions)
-
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 set(GAMMA_RELEASE_TAG "cc442ad0c5da369966cd937a96925c7b9a04e9e5")
@@ -24,4 +22,4 @@ vcpkg_install_cmake()
 vcpkg_copy_pdbs()
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/COPYRIGHT DESTINATION ${CURRENT_PACKAGES_DIR}/share/gamma RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/COPYRIGHT DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

